tests/nightly/gpu/test_bert.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import unittest
+import parlai.core.testing_utils as testing_utils
+
+
+@testing_utils.skipUnlessGPU
+class TestBertModel(unittest.TestCase):
+    """Checks that Both Biencoder and CrossEncoder of Bert can be trained
```

Comment:
Do you mind editing the docstring whitespace to be more like:

https://github.com/facebookresearch/ParlAI/blob/b412c4d88298a7c531eaab9b5e105ab5588f3a87/parlai/core/torch_agent.py#L1468-L1474